scania/scania-theme
# bug - Updated links for IE and Edge

**Describe pull-request**  
_Describe what the pull-request is about_
- Solved problem with links in IE and Edge
- Added new css variables related with links
- Refactoring

**Colors for links aren't decided, this is based on what we had in chrome**
**Please look at pull-request in SDDS for overflow and temp color for links**

**Solving issue**  
_Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)_
Fixes: #152 also related with scania/corporate-ui-site#98

**How to test**  
_Add description how to test if possible_
When you compare, look at SDDS in chrome, should look the same
1. Check links in Chrome, edge and IE
2. Check Header in all browsers
3. check Footer in all browsers
4. Check Navigation in all browsers

**Additional context**  
_Add any other context about the pull-request here._
Colors aren't decided yet, this is a setup for making it easier to switch when time comes
